editor/code/src/client.ts

```diff
@@ -90,11 +88,14 @@ export function activate(context: ExtensionContext): void {
       initializationOptions,
       markdown: { isTrusted: true, supportHtml: true },
     };
-    const serverOptions = { command: wsConfig.path, args: wsConfig.args };
+    const serverOptions: ServerOptions = {
+      command: wsConfig.path,
+      args: wsConfig.args,
+    };
 
     client = new LanguageClient(
-      "coq-lsp-server",
-      "Coq Language Server",
+      "coq-lsp",
+      "Coq LSP Server",
```

Comment:
Ditto, while I'm OK with this changes they seem unrelated to the PR title.

Reply:
It's not unrelated: `coq-lsp-server.trace.server` was being read earlier, and this was a bug: `coq-lsp.trace.server` is set, and we read that.